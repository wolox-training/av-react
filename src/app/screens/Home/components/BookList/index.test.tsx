import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import BooksList from './';

describe('BooksList test', () => {
  let container: Element | null = null;

  beforeEach(() => {
    const { container: renderContainer } = render(
      <Router>
        <BooksList />
      </Router>
    );
    container = renderContainer;
  });

  test('Should match snapshot', () => {
    expect(container).toMatchSnapshot();
  });

  test('Should render books wrapper with classname', () => {
    expect(container?.firstChild).toHaveClass('booksList');
  });
});
