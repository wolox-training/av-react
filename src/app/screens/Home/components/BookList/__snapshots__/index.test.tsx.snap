// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BooksList test Should match snapshot 1`] = `
<div>
  <div
    class="booksList"
  >
    <div
      class="sk-fade-in-half-second sk-spinner sk-three-bounce"
      color="#000"
      style="color: rgb(0, 0, 0);"
    >
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
